model/database/registry.yaml

```diff
@@ -541,9 +542,50 @@ groups:
               value: "ConsistentPrefix"
               stability: experimental
         stability: experimental
-        brief: Account or request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels).
+        brief: >
+          Effective Request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels).
+          This represents the first applicable consistency level among those specified by the request or the client.
         examples: ["Eventual", "ConsistentPrefix", "BoundedStaleness", "Strong", "Session"]
 
+  - id: registry.db.cosmosdb.network
+    type: attribute_group
+    display_name: Azure Cosmos DB Network level Attributes
+    stability: experimental
+    brief: >
+      This group defines attributes for Azure Cosmos DB Network call.
+    attributes:
+      - id: db.cosmosdb.network.host
+        type: string
+        stability: experimental
+        brief: Specifies the host address from the Gateway or Backend Service endpoint.
+        examples: ["rntbd://cdb-east-us.azure.document.com"]
+      - id: db.cosmosdb.network.port
+        type: int
+        stability: experimental
+        brief: The port used for the Gateway or Backend Service endpoint.
+        examples: 443
+      - id: db.cosmosdb.network.response.status_code
+        type: int
+        stability: experimental
+        brief: The HTTP status code returned from the network request.
+        examples: 200
+      - id: db.cosmosdb.network.response.sub_status_code
```

Comment:
is there any difference between `db.cosmosdb.network.response.sub_status_code` and `db.cosmosdb.sub_status_code` ?  Can we reuse the existing one instead of introducing a new thing?

Reply:
We want to have both, operation level statuscode and network level status code as a dimension

Reply:
I don't see `db.cosmosdb.network.response.sub_status_code` used along with `db.cosmosdb.sub_status_code` on metrics. And if they were, how can they be different on the same telemetry item? What's the difference?

Reply:
for a same telemetry item, operation level substatuscode i.e. `db.cosmosdb.sub_status_code` would indicate if network call belongs to successful/failed operation (which also indicates that it was transient/non-transient issue at network layer)

Reply:
when the network call ends you don't know how overall operation will go. Are you going to wait for it to complete before reporting telemetry?

Reply:
Yes, we are currently collecting network-level telemetry at the completion or failure of each operation. The SDK is designed to capture this information, so we already have it available. I just need to emit it.